weed/command/filer.go

```diff
@@ -239,6 +241,7 @@ func (fo *FilerOptions) startFiler() {
 		SaveToFilerLimit:      int64(*fo.saveToFilerLimit),
 		ConcurrentUploadLimit: int64(*fo.concurrentUploadLimitMB) * 1024 * 1024,
 		ShowUIDirectoryDelete: *fo.showUIDirectoryDelete,
+		DownloadSpeedInBytes:  int64(*fo.downloadMaxMBps) * 1024 * 1024,
```

Comment:
```suggestion
		DownloadMaxMBps:  int64(*fo.downloadMaxMBps) * 1024 * 1024,
```

Reply:
ok i'll check globally
But will this lead to misunderstandings, it is actually a unit of bytes